src/openforms/js/components/admin/form_design/registrations/zgw/OptionsVariablesProperties.js

```diff
@@ -0,0 +1,265 @@
+import {ValidationErrorContext} from 'components/admin/forms/ValidationErrors';
+import {FormModal} from 'components/admin/modals';
+import {ChangelistTableWrapper, HeadColumn, TableRow} from 'components/admin/tables';
+
+import {getErrorMarkup, getFieldErrors} from './utils';
+
+const EMPTY_VARIABLE_PROPERTY = {
+  variablesProperties: [{componentKey: '', eigenshap: ''}],
+};
+
+const HeadColumns = () => {
+  const intl = useIntl();
+
+  const componentText = (
+    <FormattedMessage description="Manage component column title" defaultMessage="Variable" />
+  );
+  const componentHelp = intl.formatMessage({
+    description: 'Manage component help text',
+    defaultMessage: 'The value of the selected field will be the process variable value.',
+  });
+
+  const eigenshapText = (
+    <FormattedMessage description="Manage property column title" defaultMessage="Property" />
+  );
+  const eigenshapHelp = intl.formatMessage({
+    description: 'Manage property help text',
+    defaultMessage: 'Specify a ZGW property name.',
+  });
+
+  return (
+    <>
+      <HeadColumn content="" />
+      <HeadColumn content={componentText} title={componentHelp} />
+      <HeadColumn content={eigenshapText} title={eigenshapHelp} />
+    </>
+  );
+};
+
+const VariableProperty = ({
+  index,
+  componentKey = '',
+  eigenshap = '',
+  componentFilterFunc,
+  onChange,
+  onDelete,
+}) => {
+  const intl = useIntl();
+
+  const confirmDeleteMessage = intl.formatMessage({
+    description: 'ZGW properties with variables connection delete confirmation message',
+    defaultMessage: 'Are you sure you want to remove this connection?',
+  });
+
+  return (
+    <TableRow index={index}>
+      <div className="actions actions--horizontal actions--roomy">
+        <DeleteIcon onConfirm={onDelete} message={confirmDeleteMessage} />
+      </div>
+
+      <ComponentSelection
+        name="componentKey"
+        value={componentKey}
+        onChange={onChange}
+        filter={componentFilterFunc.bind(null, componentKey)}
+      />
+      <TextInput
+        name="eigenshap"
+        value={eigenshap}
+        onChange={onChange}
+        placeholder={componentKey}
+      />
+    </TableRow>
+  );
+};
+
+VariableProperty.propTypes = {
+  index: PropTypes.number.isRequired,
+  componentKey: PropTypes.string,
+  eigenshap: PropTypes.string,
+  componentFilterFunc: PropTypes.func.isRequired,
+  onChange: PropTypes.func.isRequired,
+  onDelete: PropTypes.func.isRequired,
+};
+
+const SelectVariablesProperties = ({variablesProperties = [], onChange, onAdd, onDelete}) => {
+  const formContext = useContext(FormContext);
+  const allComponents = formContext.components;
+
+  const usedComponents = variablesProperties
+    .filter(connection => connection.componentKey !== '')
+    .map(connection => connection.componentKey);
+
+  const filterFunc = (componentKey, {key}) => componentKey === key || !usedComponents.includes(key);
```

Comment:
Is this present so that you can't map the same variable to multiple eigenschappen? Is it needed?

Reply:
Yes, we said for now we will assume one variable can be connected to one property.